QUARKUS-1223.md

```diff
@@ -28,30 +29,41 @@ Testing should include following topics:
-- Integration tests in Quarkus upstream 
-  - https://github.com/quarkusio/quarkus/tree/main/integration-tests/hibernate-search-orm-elasticsearch-coordination-outbox-polling (tests disabled by default)
+Solid coverage exists in upstream resources for most of the outlined topics.  
+All tests use Hibernate Search in combination with RESTEasy Classic: `quarkus-resteasy`, `quarkus-resteasy-jackson`, `quarkus-resteasy-jsonb`.  
+Combinations with databases: `quarkus-jdbc-postgresql`, `quarkus-jdbc-h2`.  
+
+(Following list mentions only features not present in other resources.)  
+- Integration tests in Quarkus upstream
+  - https://github.com/quarkusio/quarkus/tree/main/extensions/hibernate-search-orm-elasticsearch/deployment/src/test/java/io/quarkus/hibernate/search/elasticsearch/test
+    - Multiple persistence units
+  - https://github.com/quarkusio/quarkus/tree/main/integration-tests/hibernate-search-orm-elasticsearch-coordination-outbox-polling
     - Outbox polling
   - https://github.com/quarkusio/quarkus/blob/main/integration-tests/hibernate-search-orm-elasticsearch-tenancy
     - Multi-tenancy using separate schema
-  - https://github.com/quarkusio/quarkus/tree/main/integration-tests/hibernate-search-orm-elasticsearch (tests disabled by default)
+  - https://github.com/quarkusio/quarkus/tree/main/integration-tests/hibernate-search-orm-elasticsearch
     - Dev Services
     - Embedded indexes using `@IndexedEmbedded`
-  - https://github.com/quarkusio/quarkus/blob/main/integration-tests/hibernate-search-orm-opensearch (tests disabled by default)
+  - https://github.com/quarkusio/quarkus/blob/main/integration-tests/hibernate-search-orm-opensearch
     - OpenSearch backend
 - Quickstarts: https://github.com/quarkusio/quarkus-quickstarts/tree/main/hibernate-search-orm-elasticsearch-quickstart
     - Initial indexing using `MassIndexer`
 
 What is missing:
-- Multiple persistence units
-- Use of `@GenericField`
-- Use of Bridges
+- Use of RESTEasy Reactive.
+- Use of other supported databases.
 
 ## Automated test development
-- Integrate Quickstart (https://github.com/quarkusio/quarkus-quickstarts/tree/main/hibernate-search-orm-elasticsearch-quickstart) into testing pipelines.
-- Adopt Quarkus integration tests into QE test suite or make sure the integration tests are part of upstream testing pipeline.
-- Implement test coverage in QE test suite for missing topics (multiple PUS, `@GenericField`, Bridges).
+- Enable relevant Quickstart module in Quickstart acceptance job: https://github.com/quarkusio/quarkus-quickstarts/tree/main/hibernate-search-orm-elasticsearch-quickstart.
+- Implement test coverage in QE test suite:
+  - OpenShift scenario.
+  - Combination with extensions:
+    - RESTEasy Reactive counterparts of RESTEasy extensions used in upstream tests.
+    - Other supported databases: `jdbc-derby`, `jdbc-mariadb`, `jdbc-mssql`, `jdbc-mysql`, `jdbc-oracle` (module `sql-db/sql-app` might be a good target location).
```

Comment:
Please do not mix stuff with `sql-db/sql-app` and create new module. It's says Hibernate Search but it doesn't have too much in common with vanilla Hibernate ORM.

I would pick 2PUs: `jdbc-mariadb` and  `jdbc-oracle`, PostgreSQL is covered in upstream.